### ISSUE TITLE
Wrong file paths?

### ISSUE THREAD

**Issue by nerdmed** [teamOnHQ/tinymce#2]
I get a lot of issues when i try to load some plugins or the package at all i get this kind of errors.
This path seems totaly wrong as packages do not serve files from here. Maybe there is a way to load themes and plugins from the CDN and remove them from the package?

    Resource interpreted as Stylesheet but transferred with MIME type text/html:     "http://localhost:3000/packages/skins/lightgray/skin.min.css".

**Comment by boldkhuu** [comment]
@nerdmed Thanks for your comment, Ahmed. All skin related css files is now added by assets and required on plugin initialization as `skin_url` property. If someone needs to change the default skin, he/she can simply change this property. I didn't delete skin related files because of development without internet connection.

**Comment by nerdmed** [comment]
@boldkhuu thanks a lot. As i ran into some more issues when using tinymce with a reactive title and blaze i wrote up a simple block helper to ease things out a little bit. Maybe you have some feedback for this?! [TinyMce Helper](https://github.com/nerdmed/tinymce-helper) in the docs you will find an example project.

**Comment by boldkhuu** [comment]
@nerdmed I cloned and viewed your example repo. Which issues do you have? Is reactiveness doesn't work?

**Comment by nerdmed** [comment]
@boldkhuu honestly its very tricky. I have stopped using tinymce as it is missing some essential hooks to work correctly alongside other templating engines. They have implemented there own dom manipulation stuff and its really not well documented or easy to isolate in the code. You will get doubled nodes as soon as you edit in inline mode and:

1. delete the original string via tinymce
2. Write a new string -> TinyMce inserts to the dom 
3. Save the changes
4. Blaze will also insert to them DOM if there was reactive content 

There is no way to tell TinyMce to not update the underlying DOM after editing. That would be a correct solution. Other implementations like the angular integration had also to work around this issue caused by tiny. I ended up using x-editable as it has a lot cleaner API to control what the plugin is doing.

Hope that helps. If you are interested in using the helper template, i would be glad to add you as a contributor.
